### PR TITLE
[gfycat] Add ru/ to regex

### PR DESCRIPTION
**Pull request by LameLemon**
### Before submitting a *pull request* make sure you have:
- [X] At least skimmed through [adding new extractor tutorial](https://github.com/ytdl-org/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/ytdl-org/youtube-dl#youtube-dl-coding-conventions) sections
- [X] [Searched](https://github.com/ytdl-org/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [X] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [X] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [X] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [X] Bug fix
- [ ] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information
Gfycat urls with ru/ in the path are now properly parsed. This pull request closes issue #21779

### CODE REVIEW

File: youtube_dl/extractor/gfycat.py [hunk 1]
```diff
@@ -54,7 +71,6 @@ class GfycatIE(InfoExtractor):
 
     def _real_extract(self, url):
         video_id = self._match_id(url)
-
```
Comment (dstftw) [inline youtube_dl/extractor/gfycat.py]: Remove unrelated changes.

File: youtube_dl/extractor/gfycat.py [hunk 2]
```diff
@@ -44,6 +44,23 @@ class GfycatIE(InfoExtractor):
             'categories': list,
             'age_limit': 0,
         }
+    }, {
+        'url': 'https://gfycat.com/ru/RemarkableDrearyAmurstarfish',
```
Comment (dstftw) [inline youtube_dl/extractor/gfycat.py]: only_matching

**Comment by LameLemon** [comment]
Made the two requested changes.
